llvm_cm/test/lit.cfg

```diff
@@ -0,0 +1,19 @@
+import os
+
+import lit.formats
+
+from lit.llvm import llvm_config
+
+config.name = "llvm-cm"
+config.test_format = lit.formats.ShTest(execute_external=False)
+
+config.suffixes = [".s", ".test"]
+
+config.test_source_root = os.path.dirname(__file__)
+config.test_exec_root = config.obj_root
+
+llvm_config.use_default_substitutions()
```

Comment:
is there a way to inherit substitutions the llvm test cfg.in makes?

Reply:
Not from what I can tell. It just uses the `lit.cfg` at the root of the test directory along with `lit.site.cfg` for subdirectories, and since we're not in `llvm/test`, we don't get to take advantage of that.